test/unit/metrics/test_group_sklearn_wrappers.py

```diff
@@ -2,46 +2,187 @@
 # Licensed under the MIT License.
 
 import pytest
+import numpy as np
+import sklearn.metrics as skm
 
 import fairlearn.metrics as metrics
 
 # ======================================================
 
+a = "a"
+b = "b"
+c = "c"
+
 Y_true = [0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1]
 Y_pred = [1, 0, 1, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1]
+
+Y_t3 = [a, b, c, c, c, b, b, b, c, c, a, a, a, a, a, b, c, c]
```

Comment:
I'm assuming t is short for true, and p for pred? Can we not collapse that?

Reply:
Fair point. Done